src/winetricks

```diff
@@ -16029,6 +16029,85 @@ load_openwatcom()
 
 #----------------------------------------------------------------
 
+w_metadata origin apps \
+    title="EA Origin" \
+    publisher="EA" \
+    year="2011" \
+    media="download" \
+    file1="OriginSetup.exe" \
+    installed_file1="${W_PROGRAMS_X86_WIN}/Origin/Origin.exe" \
+    homepage="https://www.origin.com/"
+
+helper_origin_dl()
+{
+    # Skipping checksum as this changes too often
+    w_download_to origin https://origin-a.akamaihd.net/Origin-Client-Download/origin/live/OriginSetup.exe
+}
+
+load_origin()
+{
+    # Need to force wine-6.0 as Origin doesn't run below WineCX21 (wine32on64)
+    if [ "$(uname -s)" = "Darwin" ] && w_wine_version_in ,6.0 ; then
+        w_die "${W_PACKAGE} requires wine version 6.0 (or newer)"
+    fi
+
+    if [ "${WINETRICKS_FORCE}" = 0 ] && w_workaround_wine_bug 44691 "Installer fails under wine, manually unpacking it instead" 6.7,; then
```

Comment:
As many custom wine packages include a patch that fixes this but this was tweaked to allow bypassing the unpacking in favour or running the actual installer.

Reply:
Changed this to `"${WINETRICKS_FORCE}" != 1`